src/scripts/main.js

```diff
@@ -1,3 +1,77 @@
 'use strict';
 
 // write code here
+
+const url = 'https://mate-academy.github.io/phone-catalogue-static/api/phones';
+
+function phoneId() {
+  return fetch(`${url}.json`)
+    .then(response => response.json())
+    .then(phone => phone.map(phones => phones.id));
+};
+
+function getFirstReceivedDetails(oneId) {
+  const phonesId = oneId.map(firstId => {
```

Comment:
```suggestion
  const phonesIds = idsArray.map(id => {
```
Тут получится массив  с id, и тебе приходит массив id, а не один id